pkg/cosign/env/env.go

```diff
@@ -0,0 +1,120 @@
+//
+// Copyright 2022 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package env
+
+import (
+	"fmt"
+	"os"
+	"sort"
+	"strings"
+)
+
+type Variable string
+
+type VariableOpts struct {
```

Comment:
nit: types might be nice here so users know to pass `=1` or `=true` vs. `=my password`. You could do this directly in the descriptions too.

Reply:
Great idea, will add it!